### ISSUE TITLE
Investigators vs. Fundee

### ISSUE THREAD

**Issue by miketaormina** [AllenNeuralDynamics/aind-data-transfer-models#64]
@camilolaiton @jtyoung84 
Right now, it looks like the list of Investigators in data_description is built by parsing the `fundee` string in the Funding model, but there are typically additional names that should be placed in the `Investigators` entry. I've asked Saskia for some clarification on these fields and verified that they do not need to be the same. I do think, however, that the names from the funding belong in the list of investigators by default.

### Example:
`fundee: "Director O'Institute, Ian A. Projectlead"` (this is not modeled as a list in data-schema, just a string)
```
investigators: [
    "Director O'Institute",
    "Ian A. Projectlead",
    "Justin Theelab",
    "Reese Arch Assistant"
]
```

### Method for Inclusion
At least for SmartSPIM, we already have the list of investigators in metadata for the raw data. Similar to specifying the institute at the time of job configuration, we could possibly add it to the configuration here:
https://github.com/AllenNeuralDynamics/aind-data-transfer-models/blob/ad888586e797c6235ff0a44329ea6ba5d484ee1b/src/aind_data_transfer_models/core.py#L560-L564

But I'm not sure if continuing to add fields to check for here is the right way to go, or if it makes a mess of if/then statements. Another problem would be that the list of investigators would need some additional logic to build from both inputs, as it should be the Union of the fundees and the list of investigators from a project's request form.

**Comment by miketaormina** [comment]
On second thought, maybe AllenNeuralDynamics/aind-metadata-mapper is where this issue belongs?

Here is where the investigators list is built from the funding model:

https://github.com/AllenNeuralDynamics/aind-metadata-mapper/blob/53a9cdf30ce18ddd1cf006a2ae46bea5311c20e4/src/aind_metadata_mapper/gather_metadata.py#L175-L191

**Comment by camilolaiton** [comment]
@miketaormina I think the metadata mapper is the right place since it is being called in the new data transfer service. Is that right @jtyoung84 ?

**Comment by jtyoung84** [comment]
Yeah. Users can pass in a settings dictionary that will be used in aind-metadata-mapper to create a data_description file. I think the code might need to be updated to add custom investigators though.

**Comment by camilolaiton** [comment]
> Yeah. Users can pass in a settings dictionary that will be used in aind-metadata-mapper to create a data_description file. I think the code might need to be updated to add custom investigators though.

Do you know how much time will it take? This impacts our ability to send emails to investigators that are waiting for their datasets to be processed.

**Comment by miketaormina** [comment]
I'm not sure what is meant by "custom investigators," does that imply that the metadata service has a list of investigators somewhere? Like, the mapper needs a return for something like `aind-metadata-service/investigators/Camilo Laiton/`?

**Comment by jtyoung84** [comment]
@dyf How do you want to prioritize this?

**Comment by jtyoung84** [comment]
> I'm not sure what is meant by "custom investigators," does that imply that the metadata service has a list of investigators somewhere? Like, the mapper needs a return for something like `aind-metadata-service/investigators/Camilo Laiton/`?

right now, the investigators are pulled from a SmartSheet. I can add an option for users to add their own configs when requesting a data upload.

**Comment by dyf** [comment]
investigators are associated to projects. we should just update the smartsheet if certain projects are incorrect.

**Comment by jtyoung84** [comment]
Is there a distinction between fundees and investigators in the SmartSheet? Currently, the data_description pulls from the list of fundees to build the investigators list.

**Comment by dyf** [comment]
The smartsheet has one row per funding source / project pair.

In the schema, `data_description.investigators` should be the union of all the investigators on the project's funding sources.

A single funding source's fundee list should just be the investigators from the single row of the spreadsheet.

@saskiad Is that right?

**Comment by saskiad** [comment]
technically, no. Fundees could include trainees (etc) who are not "investigators". I think the distinction is pretty minor though.

**Comment by saskiad** [comment]
- [x] Update Funding smartsheet to have an investigator column
- [x] Update ~transfer~ metadata service to pull this column
- [x] Use metadata ~service~ mapper to create data description

**Comment by camilolaiton** [comment]
Hello @saskiad, are there any updates about this?

**Comment by saskiad** [comment]
No updates yet

**Comment by saskiad** [comment]
The smartsheet has been updated to (a) separate the  Investigator and Fundee columns and (b) allow for subprojects within existing projects where there might be different investigators for different parts of the project. (e.g. the U19 has several subprojects). It might be worth taking a look to see what this looks like for your projects and let us know what needs to be updated.
@jtyoung84 can you update the transfer service to pull the investigator separately from the fundee?

**Comment by jtyoung84** [comment]
I'll put it on the ToDo list. It shouldn't be too hard.

**Comment by jtyoung84** [comment]
@saskiad . I deployed this change to production. However, the investigators in most of the projects in the Smartsheet are blank.

**Comment by saskiad** [comment]
@camilolaiton @miketaormina Which projects/subprojects are you working with mostly? We've updated the smartsheet structure so we should make sure we have the investigator information you want in that sheet.

**Comment by miketaormina** [comment]
I can add names to the Smartsheet for projects on our tracker, if that's how it needs to be tracked. I'll check with Sharon and Bargavi to make sure I get the correct names onto projects. Right now, each specimen has one specific requestor, but one project might have specimens from multiple investigators. I've been putting each specimen's requestor name in the processing manifest and continue to do that as well.

**Comment by saskiad** [comment]
@miketaormina can you get this project information so we can get it into the Smartsheet? There are restrictions on who can edit it because it impacts too many things, so I need you to send me (or Shelby) the information.

**Comment by miketaormina** [comment]
Sorry I missed this @saskiad . I'm pretty sure that all of the common AIBS projects are already on the sheet. We generally use
- BG AAV Toolbox
- CTY GEnetic Tools
- Thalamus in the middle - Project 6 Molecular Science Core
